lib/LaTeXML/resources/RelaxNG/LaTeXML-structure.rnc

```diff
@@ -21,7 +21,7 @@ element document { document_attributes, document_model }
 
 ## The content allowable as the main body of the document.
 document.body.class = Para.model | paragraph | subsubsection
-  | subsection | section | chapter | part | slide | sidebar
+  | subsection | section | chapter | part | frame | sidebar
```

Comment:
This change will require even "simple" slides (eg. Classic LaTeX slides) to have each slide wrapped in an `ltx:frame`.  Would probably be better to use `slide | frame`.

Except that `frame` is way too generic and overloaded for the schema.  Perhaps `slide-sequence` would be better?

Reply:
I used `slide` and `frame` to stay in line with the beamer terminology.
At first glance I don't like `slide-sequence`, it seems to verbose. 
I'll see if I can come up with a better term.